### ISSUE TITLE
Suggestions for a stylish make-over for "Video details" page

### ISSUE THREAD

**Issue by oas777** [elan-ev/tobira#946]
https://tobira.opencast.org/~manage/videos/EJic0mSes6f

- This page has quite a number of different fonts, font sizes, italics, and bold types. 
- Top-down, it mixes (editable and non-editable) metadata (Created, Updated, Title...) with a clickable symbols/feature ("Open in video editor"). 
- "Share via direct link:..." should become a clickable "Share" like the one on the video page https://tobira.opencast.org/lectures/d-infk/2019/autumn/252-0856-00L/v/EJic0mSes6f 
![grafik](https://github.com/elan-ev/tobira/assets/1752342/a032fefe-bcd0-425a-8297-9e6f94fc42ef)

In combination, I see "Open in video editor" and "Share" at the top, and a couple of blocks below:

- Metadata (the tree or four elements we defined for quick editing)
- Access policy
- Statistics
- Misc (for "Referencing pages" and some other [information from the technical page](https://github.com/elan-ev/tobira/issues/807#issuecomment-1731903438))

**Comment by LukasKalbertodt** [comment]
I agree that this page could use a make-over. I'd suggest tackling that during/after we implement editable metadata (#371).
